include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1534,14 +1534,11 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::do_reinit()
   if (fast_path && !polynomials_are_hat_functions)
     {
       const std::size_t n_batches =
-        n_q_points_scalar / n_lanes_internal +
-        (n_q_points_scalar % n_lanes_internal > 0 ? 1 : 0);
+        (n_q_points_scalar + n_lanes_internal - 1) / n_lanes_internal;
```

Comment:
What you're doing here is rounding up `n_q_points_scalar/n_lanes_internal`. Can you put this into a comment?